akka-stream/src/main/scala/akka/stream/impl/PhasedFusingActorMaterializer.scala

```diff
@@ -395,12 +396,13 @@ case class PhasedFusingActorMaterializer(
       PhasedFusingActorMaterializer.DefaultPhases)
   }
 
-  def materialize[Mat](
-    graph:             Graph[ClosedShape, Mat],
-    subflowFuser:      GraphInterpreterShell ⇒ ActorRef,
-    initialAttributes: Attributes,
-    defaultPhase:      Phase[Any],
-    phases:            Map[IslandTag, Phase[Any]]): Mat = {
+  // TODO can I expose this or not?
+  override def materialize[Mat](
+    graph:                 Graph[ClosedShape, Mat],
+    NOT_USED_subflowFuser: GraphInterpreterShell ⇒ ActorRef, // FIXME not used here?
```

Comment:
I don't think it is needed here. It was needed before because the materializer was doing the subfusing hence it had to know about it. Since now it is factored out into a Phase, the materializer itself does not need to be aware of this anymore.